targets/compose/firebase.go

```diff
@@ -0,0 +1,33 @@
+package compose
+
+var (
+	// FirebaseService represents a docker compose firebase service.
+	FirebaseService = Service{
+		Image:   "wceolin/firebase-emulator",
```

Comment:
@oliverpauffley Would there be benefit to building and hosting this image ourselves?

We would have greater control over its maintenance, as there have been no updates for over a year (it's essentially only installing a CLI via npm anyway).

What do you reckon? 🙂